pkgs/development/interpreters/tcl/generic.nix

```diff
@@ -61,6 +61,8 @@ let
         "--enable-man-symlinks"
         # Don't install tzdata because NixOS already has a more up-to-date copy.
         "--with-tzdata=no"
+        # configure check broke due to GCC 14
+        "ac_cv_header_stdc=yes"
```

Comment:
`lib.versionOlder version "9.0"` applies it also to 8.6, use `lib.versionOlder version "8.6"` instead